.github/workflows/end2end-test-playwright.yml

```diff
@@ -39,22 +39,22 @@ jobs:
 
             - name: Install Playwright dependencies
               run: |
-                  npx playwright install chromium --with-deps
+                  npx playwright install chromium firefox webkit --with-deps
```

Comment:
It's a shame there isn't a better summary of the browsers in the test output. Dividing the test results by browser would be nice. In CI we can at least see that two tests were skipped (the firefox ones), so that validates that the tests are running in FF.

I suppose it can't be helped if that's how playwright does it. 😄 

Surprisingly it's also not clear in the more verbose mode when run locally:
```
  ✓  1 …Inserting blocks (@firefox, @webkit) › inserts blocks by dragging and dropping from the global inserter (5s)
  ✓  2 …erting blocks (@firefox, @webkit) › cancels dragging blocks from the global inserter by pressing Escape (5s)
  ✓  3 …serting blocks (@firefox, @webkit) › inserts patterns by dragging and dropping from the global inserter (3s)
  ✓  4 …ting blocks (@firefox, @webkit) › cancels dragging patterns from the global inserter by pressing Escape (3s)
  ✓  5 …Inserting blocks (@firefox, @webkit) › inserts blocks by dragging and dropping from the global inserter (7s)
  ✓  6 …erting blocks (@firefox, @webkit) › cancels dragging blocks from the global inserter by pressing Escape (5s)
  ✓  7 …serting blocks (@firefox, @webkit) › inserts patterns by dragging and dropping from the global inserter (4s)
  ```

Reply:
Here's the full output of the test results:

```
Running 8 tests using 1 worker

  ✓  1 [webkit] › editor/various/inserting-blocks.spec.js:21:2 › Inserting blocks (@firefox, @webkit) › inserts blocks by dragging and dropping from the global inserter (6s)
  ✓  2 [webkit] › editor/various/inserting-blocks.spec.js:100:2 › Inserting blocks (@firefox, @webkit) › cancels dragging blocks from the global inserter by pressing Escape (6s)
  ✓  3 [webkit] › editor/various/inserting-blocks.spec.js:157:2 › Inserting blocks (@firefox, @webkit) › inserts patterns by dragging and dropping from the global inserter (5s)
  ✓  4 [webkit] › editor/various/inserting-blocks.spec.js:227:2 › Inserting blocks (@firefox, @webkit) › cancels dragging patterns from the global inserter by pressing Escape (4s)
  -  5 [firefox] › editor/various/inserting-blocks.spec.js:21:2 › Inserting blocks (@firefox, @webkit) › inserts blocks by dragging and dropping from the global inserter
  ✓  6 [firefox] › editor/various/inserting-blocks.spec.js:100:2 › Inserting blocks (@firefox, @webkit) › cancels dragging blocks from the global inserter by pressing Escape (8s)
  -  7 [firefox] › editor/various/inserting-blocks.spec.js:157:2 › Inserting blocks (@firefox, @webkit) › inserts patterns by dragging and dropping from the global inserter
  ✓  8 [firefox] › editor/various/inserting-blocks.spec.js:227:2 › Inserting blocks (@firefox, @webkit) › cancels dragging patterns from the global inserter by pressing Escape (5s)
```

There is the browser/project name in front of each test case. Just that it won't show them if the terminal isn't wide enough. I think Playwright should at least show the project name and just truncate the middle part though, but that's up to them.